Dockerfile

```diff
@@ -11,7 +11,7 @@ RUN export DEBIAN_FRONTEND=noninteractive \
         xvfb
 
 # Install OBS
-ARG OBS_STUDIO_VERSION=27.1.0
+ARG OBS_STUDIO_VERSION=27.1.3
```

Comment:
Can you use something like `27.1.+`?

Reply:
Yes, however this would hardcode the regex...
Wait, it can be overridden, it's an ARG . -___-

Reply:
See https://github.com/obs-websocket-community-projects/obs-websocket-docker/pull/9 @ChristopheCVB